examples/cms/src/FieldMapping.tsx

```diff
@@ -0,0 +1,223 @@
+import type { ManagedCollection, ManagedCollectionField } from "framer-plugin"
+import type { DataSource, FieldIds, ManagedCollectionFields } from "./data"
+
+import { framer } from "framer-plugin"
+import { useState, useMemo, useEffect, memo } from "react"
+import { computeFieldsFromDataSource, mergeFieldsWithExistingFields, syncCollection } from "./data"
+import { Spinner } from "./components/Spinner"
+
+function ChevronIcon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="8" height="16">
+            <path
+                d="M 3 11 L 6 8 L 3 5"
+                fill="transparent"
+                strokeWidth="1.5"
+                stroke="#999"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+            />
+        </svg>
+    )
+}
+
+interface FieldMappingRowProps {
+    field: ManagedCollectionField
+    isIgnored: boolean
```

Comment:
Nit. When creating a variable in a `let` or `const` I like to prefix the variable by something like _is_ or _has_. But for property names I don't. That's because native HTML elements do that neither. It's `disabled` and `readonly`, and not `isDisabled` and `isReadOnly`

Reply:
Btw. Currently the Props are named `isIgnored` and `onFieldToggle`. Those two are related when you read the implementation but naming wise they seem unrelated

Reply:
I wonder if the term `enabled` or `disabled` is easier to understand btw

Reply:
Indeed, I'll rename to `disabled`. Keeping in mind that in future iteration we can have `unsupported` fields too.